### PR TITLE
Run tests on NodeJS 19.6.0

### LINKED ISSUE #4710: Test fails on NodeJS 19.6.0
### Describe the problem and steps to reproduce it:

1. Install NodeJS 19.6.0
2. Install this package, run tests
3. Observe that one test fails

### What happened?
```
 FAIL  tests/unit/parsers/test.json.js
  ● JSONParser › should show a message if bad JSON

    expect(received).toContain(expected) // indexOf

    Expected substring: "Unexpected token b"
    Received string:    "Unexpected token 'b', \"blah\" is not valid JSON"

      18 |     expect(errors[0].code).toEqual(messages.JSON_INVALID.code);
      19 |     expect(errors[0].message).toContain('Your JSON is not valid.');
    > 20 |     expect(errors[0].description).toContain('Unexpected token b');
         |                                   ^
      21 |   });
      22 | });
      23 |

      at Object.toContain (tests/unit/parsers/test.json.js:20:35)
```
### What did you expect to happen?
Tests pass.
### Anything else we should know?
This change apparently happened recently, some time between v18.11.0 and 19.6.0. Tested on https://npm.runkit.com/
Modern Chromium browsers also produce error `"Unexpected token 'b', \"blah\" is not valid JSON"`

### PR DESCRIPTION
**Pull request by bershanskiy**
Fixes #4710.

Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.

Please delete anything that isn't relevant to your patch.

- [x] This PR relates to an existing open issue and there are no existing PRs open for the same issue.
- [x] Add `Fixes #ISSUENUM` at the top of your PR.
- [x] Add a description of the changes introduced in this PR.
- [x] The change has been successfully run locally.
- Add tests to cover the changes added in this PR. - N/A

Once you have met the above requirements please replace this section with a `Fixes #ISSUENUM` linking to the issue fixed by this PR along with an explanation of the changes. Thanks for your contribution!

### CODE REVIEW

File: tests/unit/parsers/test.json.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ describe('JSONParser', () => {
     expect(errors.length).toEqual(1);
     expect(errors[0].code).toEqual(messages.JSON_INVALID.code);
     expect(errors[0].message).toContain('Your JSON is not valid.');
-    expect(errors[0].description).toContain('Unexpected token b');
+    expect(errors[0].description).toContain('Unexpected token');
```
Comment (willdurand) [inline tests/unit/parsers/test.json.js:20]: Could you please use https://jestjs.io/docs/expect#tomatchregexp--string instead? It'd be good to verify that `blah` (or just `b` like before) is part of the message.

**Review by willdurand — CHANGES_REQUESTED** [review]
Thanks for working on this!

**Comment by willdurand** [comment]
While you are at it, could you please update the Circle config to run Node 19 instead of Node 18?

**Comment by bershanskiy** [comment]
Thanks for the review, I incorporated the changes.

**Review by willdurand — APPROVED** [review]
Thank you!

**Pull request merged**